cmd/kes/update.go

```diff
@@ -80,6 +80,11 @@ func updateInplace() error {
 		return err
 	}
 
+	if rel == version {
+		fmt.Printf("You are already running the most recent version of 'kes'.\n")
```

Comment:
```suggestion
		fmt.Printf("You are already running the latest version %q.\n", version)
```

Eventually, tell me what the latest version is 😃